blog-post.html

```diff
@@ -2,11 +2,92 @@
+        </div>
+      </div>
+
+      <div id="footer-container-bottom">
+        <div class="footer-left-column">
+          <div class="footer-content">
+            <div class="name-email">
+              <input id="name" name="name" rows="1" cols="5" type="text" placeholder="Name"/>
+            &nbsp;
+              <input id="email" name="email" rows="1" cols="5" type="text" placeholder="Email"/>
+            </div>
+          <br>
+          <br>
+            <div class="message">
+              <textarea id="message" name="message" rows="10" cols="20" placeholder="Message"></textarea>
+            <br>
+              <button class="button4" type="button"><img class="mail-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-email.png"/>SEND MESSAGE</button>
+            <br>
+            </div>
+          </div>
+        </div>
+
+        <!--<div class="footer-right-column"></div>-->
+        <div class="footer-right-column">
+          <div class="footer-right-column-top">
+            <p class="comments-desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam consequatur quod dignissimos, ex nihil quis. Eum, similique! Maiores accusamus dolorum ad sit fuga maxime harum!
+            </p>
+          </div>
+         <!--<div class="footer-right-column">-->
+          <div class="footer-right-column-left">
+            <p class="home-desc"><img class="home-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-home.png"/> 1234 Somewhere Road<br>Burlington, VT 00000<br>USA</p>
+            <p class="phone-desc"><img class="phone-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-phone.png"/> (000) 000-0000</p>
+            <p class="email-desc"><img class="email-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-email.png"/> info@bca.com</p>
+          </div>
+
+          <div class="footer-right-column-right">
+            <p class="twitter-desc"><img class="twitter-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-twitter.png"/> @untitled</p>
+            <p class="insta-desc"><img class="insta-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-instagram.png"/> @instagram</p>
+            <p class="dribbble-desc"><img class="dribbble-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-dribbble.png"/> dribbble.com</p>
+            <p class="facebook-desc"><img class="facebook-img" src="file:///Users/jaysondurante/Desktop/Jayson/Burlington%20Code%20Academy/Remock%20Proj%20Pics/footerlink-facebook.png"/> facebook.com</p>

```

Comment:
all of these images are Local to your computer. Try using ones that are hosted somewhere